monitor.c

```diff
@@ -485,7 +485,10 @@ static int read_and_act(struct active_array *a, fd_set *fds)
 		 * wasn't requested, transition to read-auto.
 		 */
 		char buf[64];
-		read_attr(buf, sizeof(buf), a->metadata_fd);
+		int n = read_attr(buf, sizeof(buf), a->metadata_fd);
+
+		if (n <= 0)
```

Comment:
shouldn't we act like it should be readonly if we cannot read property? I would not stop mdmon in case of we cannot read the property.

I hope it is only theoretical problem..